modules/andes-core/client/src/main/java/org/wso2/andes/client/protocol/AMQProtocolHandler.java

```diff
@@ -575,7 +575,7 @@ public void run()
         });
         if (PROTOCOL_DEBUG)
         {
-            _protocolLogger.debug(String.format("SEND: [%s] %s", this, frame));
+            _protocolLogger.info(String.format("SEND: [%s] %s", this, frame));
```

Comment:
we're still checking PROTOCOL_DEBUG so wouldn't it print this only if PROTOCOL_DEBUG is set to true?